### PR TITLE
PMM-7395 Adding Contribution Guidelines

### PR DESCRIPTION
**Pull request by denisok**
[PMM-7395](https://jira.percona.com/browse/PMM-7395) Add contribution guidelines to PMM GitHub - intro/explain on folders/repo

Co-authored-by: Nurlan Moldomurov <nurlan.moldomurov@percona.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+An excellent resource is [Elika Etemad's article on filing good bug reports.](http://fantasai.inkedblade.net/style/talks/filing-good-bugs/).
+
+As a general rule of thumb, please try to create bug reports that are:
+
+- *Reproducible.* Include steps to reproduce the problem.
+- *Specific.* Include as much detail as possible: which version, what environment, etc.
+- *Unique.* Do not duplicate existing tickets.
+
+## Setup your local development environment
+<explanation of local setup>
```
Comment (BupycHuk) [inline CONTRIBUTING.md:87]: here we can say that user can clone pmm-managed, run `make env-up` to start PMM Server and then clone `pmm-agent` and run `make setup-dev` there, so he will have a ready environment with minimum stuff required.

**Comment by denisok** [comment]
@fabio-silva we miss UI part on how to contribute to UI.
@puneet0191 maybe there are something we should add about E2E

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+### Exporters
+
+Exporters by itself is an independent applications, so each of them contains their own README files explaining how to contribute [see PMM Client](#PMM-Client).
+
+### UI
+
+See [Grafana Dashboards Contribution Guide](https://github.com/percona/grafana-dashboards/blob/main/CONTRIBUTING.md).
+
+< TODO:Explain how to contribute to Grafana >
```
Comment (BupycHuk) [inline CONTRIBUTING.md:112]:
```suggestion

```
I think we can remove it.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+* code can only be merged via PRs
+* at least 2 reviews are assigned and they should have approved the PR
+* tests and checks passed
+* FB was tested by QA (if it applies)
+
+If above criteria are met - anyone (PR owner or any other developer) can merge the code.
+
+In case of CODEOWNERs they would merge it.
+
+If there are any issues with tests, checks or CODEOWNER is absent but there are 2 reviews - PR owner should ask anyone from PMM Admins to merge PRs.
```
Comment (BupycHuk) [inline CONTRIBUTING.md:153]: we have a separate block for merging rules in GIT_AND_GITHUB.md, should we duplicate it here?

**Review by BupycHuk — DISMISSED** [review]

**Review by BupycHuk — DISMISSED** [review]

File: GIT_AND_GITHUB.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+## Preparation
+
+Make sure your `git user.name` is set, and `user.email` is set to your corporate (if you are Perconian) or personal email, i.e. john.doe@percona.com.
+
+Use topic branches in the same repository instead of GitHub forks if you have such permissions.
+
+Name topic branches like this: `PMM-1234-short-description`. Always start with `PMM-XXXX` or `SAAS-XXXX`. Never omit a short description. Separate words with dashes, not underscore. Use only lowercase letters.
+
+If you are community contributor and there is no ticket exists, either create one or just use `short-description` style for your forked repo branch.
```
[the remark this answers is not in the document]
Comment (denisok) [reply GIT_AND_GITHUB.md:15]: depends on a repo, for https://github.com/percona/mongodb_exporter/pull/474 it is quite fine to not have a ticket.

[the remark this answers is not in the document]
Comment (BupycHuk) [reply GIT_AND_GITHUB.md:15]: Yeah, there it can be Github issue :)

**Comment by denisok** [comment]
@atymchuk please review

**Review by rnovikovP — APPROVED** [review]
🚀  to have this cleaned up and clarified!

**Pull request merged**